analysis/topEFT/make_lambda_plot.py

```diff
@@ -0,0 +1,216 @@
+# This script makes a plot of the energy scale Lambda
+#   - The WC limits (for 1TeV) are hardcoded in this script
+#   - The script converts these to Lambda for various assumptions for c
+#   - Plots the resulting Lambda for each WC
+
+import matplotlib.pyplot as plt
+from matplotlib.lines import Line2D
+import numpy as np
+import matplotlib.patches as mpatches
+import copy
+
+import mplhep as hep
+
+WC_LST = [
+    "cQq13", "cQq83", "cQq11", "ctq1", "cQq81", "ctq8",
+    "ctt1", "cQQ1", "cQt8", "cQt1",
+    "ctW", "ctZ", "ctp", "cpQM", "ctG", "cbW", "cpQ3", "cptb", "cpt",
+    "cQl3i", "cQlMi", "cQei", "ctli", "ctei", "ctlSi", "ctlTi"
+]
+
+WC_FORMAT_DICT = {
+    "cQq13"  : "$c_{\mathrm{Qq}}^{31}$",
+    "cQq83"  : "$c_{\mathrm{Qq}}^{38}$",
+    "cQq11"  : "$c_{\mathrm{Qq}}^{11}$",
+    "ctq1"   : "$c_{\mathrm{tq}}^{1}$" ,
+    "cQq81"  : "$c_{\mathrm{Qq}}^{18}$",
+    "ctq8"   : "$c_{\mathrm{tq}}^{8}$" ,
+    "ctt1"   : "$c_{\mathrm{tt}}^{1}$" ,
+    "cQQ1"   : "$c_{\mathrm{QQ}}^{1}$" ,
+    "cQt8"   : "$c_{\mathrm{Qt}}^{8}$" ,
+    "cQt1"   : "$c_{\mathrm{Qt}}^{1}$" ,
+    "ctW"    : "$c_{\mathrm{tW}}$"  ,
+    "ctZ"    : "$c_{\mathrm{tZ}}$"  ,
+    "ctp"    : "$c_{\mathrm{t} \\varphi}$"  ,
+    "cpQM"   : "$c_{\\varphi \mathrm{Q}}^{-}$" ,
+    "ctG"    : "$c_{\mathrm{tG}}$"  ,
+    "cbW"    : "$c_{\mathrm{bW}}$"  ,
+    "cpQ3"   : "$c_{\\varphi \mathrm{Q}}^{3}$" ,
+    "cptb"   : "$c_{\\varphi \mathrm{t b}}$" ,
+    "cpt"    : "$c_{\\varphi \mathrm{t}}$"  ,
+    "cQl3i"  : "$c_{\mathrm{Q\ell}}^{3(\ell)}$" ,
+    "cQlMi"  : "$c_{\mathrm{Q\ell}}^{-(\ell)}$" ,
+    "cQei"   : "$c_{\mathrm{Qe}}^{(\ell)}$"  ,
+    "ctli"   : "$c_{\mathrm{t\ell}}^{(\ell)}$"  ,
+    "ctei"   : "$c_{\mathrm{te}}^{(\ell)}$"  ,
+    "ctlSi"  : "$c_{\mathrm{t}}^{S(\ell)}$" ,
+    "ctlTi"  : "$c_{\mathrm{t}}^{T(\ell)}$" ,
+}
+
+################### From TOP-22-006 ###################
+
+TOP22006_LIMS_DICT = {
```

Comment:
We could think about reading a json file instead of hard coding the values, but that doesn't need to hold up this PR.

Reply:
Agreed